packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -83,20 +83,14 @@ export class DocumentDeltaConnection
 	}
 
 	public get disposed() {
-		// Increase the stack trace limit temporarily, so as to debug better in case it occurs.
-		// We are seeing this in telemetry and we are unable to figure out why it is happening, so this should help.
-		const originalStackTraceLimit = (Error as any).stackTraceLimit;
 		try {
-			(Error as any).stackTraceLimit = 50;
 			assert(
 				this._disposed || this.socket.connected,
```

Comment:
If this no longer happens in prod then I'd remove try/catch.
But if we want to keep it forever, then I'd convert assert into error log statements (and still remove try/catch :))

Reply:
Same in other places